dev-client/.env.sample

```diff
@@ -7,3 +7,4 @@ PUBLIC_MAPBOX_TOKEN=""
 SENTRY_DSN=https://xyz@o4506299756642304.ingest.sentry.io/abcd
 SENTRY_DISABLE_AUTO_UPLOAD="true"
 SENTRY_PROJECT="terraso-lpks"
+SENTRY_ENABLED="false"
```

Comment:
question: why are some of these quoted and some not?

Reply:
no idea. i quoted all of them for consistency.